ci/tox/Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN set -x \
 
 ARG python_version
 ENV PYTHON_VERSION=${python_version}
+ARG check_coverage
+ENV CHECK_COVERAGE=${check_coverage}
```

Comment:
Nit: move down since now `PYTHON_VERSION` and `PYTHON` are split

Reply:
Ok, I will also add a comment explaining what it does since it does not have any reference in this file itself, but only in entrypoint.sh